pkg/operator/ceph/disruption/clusterdisruption/mon.go

```diff
@@ -31,14 +31,8 @@ const (
 	pdbName = "rook-ceph-mon-pdb"
 )
 
-// setting minAvailable for mons at floor((n+1)/2) an logging and error if n is even
-// monCount - minAvailable
-// 1,2      - 0 (no HA)
-// even     - n -1
-// odd      - floor((n+1)/2)
 func (r *ReconcileClusterDisruption) reconcileMonPDB(cephCluster *cephv1.CephCluster) error {
```

Comment:
Does the disruption controller still need to reconcile the mon pdb? Seems like we just want the cluster reconciler to manage the mon pdb so we can avoid any conflicts between the controllers. For example, if a mon failover is in progress, we don't want the disruption controller to reconcile the pdb.

Reply:
since we changed from dynamic `minAvailable` to a static `maxUnavailable=1`, the reconciler won't change the `maxUnavailable` after creating it initially. See (https://github.com/rook/rook/blob/c2a62b63f9414396a1ba1793210c85eb871baf24/pkg/operator/ceph/disruption/clusterdisruption/static_pdb.go#L37-L48).  So disruption controller reconciler won't interfere and change the mon pdbs while mons are failing over.